apps/new-tab/src/Settings.tsx

```
@@ -65,6 +87,41 @@ function BigButton(props: any) {
+  children?: BookmarkTreeNode[];
+  dateAdded?: number;
+  dateGroupModified?: number;
+  id: string;
+  index?: number;
+  parentId?: string;
+  title: string;
+  unmodifiable?: "managed";
+  url?: string;
+}
+
+type Bookmark = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Bookmark'. Consider to delete it or rename it.

'Bookmark' is defined here:


(lint/suspicious/noRedeclare)